src/common/api-config/api.config.service.ts

```diff
@@ -972,4 +976,49 @@ export class ApiConfigService {
 
     return port;
   }
+
+  isStateChangesFeatureActive(): boolean {
+    const isStateChangesActive = this.configService.get<boolean>('features.stateChanges.enabled');
+    if (isStateChangesActive === undefined) {
+      return false;
+    }
+
+    return isStateChangesActive;
+  }
+
+  getStateChangesFeaturePort(): number {
+    const stateChangesPort = this.configService.get<number>('features.stateChanges.port');
+    if (stateChangesPort === undefined) {
+      throw new Error('No state changes port present');
+    }
+
+    return stateChangesPort;
+  }
+
+  getStateChangesUrl(): string {
+    const url = this.configService.get<string>('features.stateChanges.url');
```

Comment:
maybe we should provide a better naming, such as `features.stateChanges.rabbitUrl` ? add a fallback to `features.stateChanges.url` as well